src/components/Comments/ReplyCommentItem.js

```diff
@@ -158,16 +162,28 @@ const ReplyCommentItem = ({
       onLongPress();
     }
   };
-
   const username = comment?.data?.anon_user_info_color_name
-    ? `Anonymous ${comment.data?.anon_user_info_emoji_name}`
+    ? `${comment.data.anon_user_info_color_name} ${comment.data?.anon_user_info_emoji_name}`
     : user?.data?.username;
 
+  const onPressDm = async () => {
+    try {
+      refSheet.current.open();
+      setLoading({...loading, loadingGetAllowAnonDmStatus: true});
+      const data = await getAllowAnonDmStatus('comment', comment?.id);
+      setUserAllowDm(data?.user.allow_anon_dm);
+    } catch (e) {
+      console.log(e);
+    } finally {
+      setLoading({...loading, loadingGetAllowAnonDmStatus: false});
+    }
+  };
+
   const onPressDM = async () => {
     try {
       setLoading({...loading, loadingDm: true});
       if (!comment?.data?.anon_user_info_color_name) {
-        const channelName = [username, profile?.myProfile?.username].join(',');
+        const channelName = username;
         const selectedUser = {
           user: {
             name: channelName,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [182-196]

In the `onPressDM` function, the `sendMessageDM` function is now called with an additional parameter `feedId`. Ensure that this change doesn't break any existing functionality where this method is used.



<!-- This is an auto-generated comment by OSS CodeRabbit -->